src/pages/chaeyeon/Main/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { FooterData } from "./FooterData";
+const Footer = () => {
+  return;
+  <div>
+    {FooterData.map((value, key) => {
+      <span key={key}>{value}</span>;
+    })}
```

Comment:
컴포넌트의 네이밍 컨벤션인 PascalCase로 사용하기 때문에 가독성이 떨어짐.
이런 상수 데이터의 경우, 컨벤션마다 다르지만 보통 UPPER_SNAKE_CASE로 활용
```suggestion
    {FOOTER_DATA.map((value, key) => {
      <span key={key}>{value}</span>;
    })}
```